src/extraction/FStar.Extraction.ML.Term.fs

```diff
@@ -1438,8 +1438,14 @@ and term_as_mlexpr' (g:uenv) (top:term) : (mlexpr * e_tag * mlty) =
                              in
                              if TcEnv.debug tcenv <| Options.Other "Extraction"
                              || TcEnv.debug tcenv <| Options.Other "ExtractNorm"
-                             then let a = FStar.Util.measure_execution_time
-                                          (BU.format1 "(Time to normalize top-level let %s)" (Print.lbname_to_string lb.lbname))
+                             then let _ = BU.print2 "Starting to normalize top-level let %s)\n\tlbdef=%s"
+                                            (Print.lbname_to_string lb.lbname)
+                                            (Print.term_to_string lb.lbdef) in
+                                  // Options.set_option "debug_level"
+                                  //   (Options.List [Options.String "Norm"; Options.String "Extraction"]);
```

Comment:
I'm keeping this comment here, since it's pretty common in debugging to uncomment this and flip on an additional level of debugging